specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/azureADOnlyAuthentication.json

```diff
@@ -0,0 +1,299 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-06-01-preview",
+    "title": "SynapseManagementClient",
+    "description": "Azure Synapse Analytics APIs"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/azureADOnlyAuthentications/{azureADOnlyAuthentication}": {
+      "get": {
+        "operationId": "AzureADOnlyAuthentications_Get",
+        "summary": "Get Azure Active Directory only authentication property",
+        "description": "Gets a Azure Active Directory only authentication property",
+        "tags": [
+          "WorkspaceAzureADOnlyAuthentications"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/azureADOnlyAuthenticationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/AzureADOnlyAuthentication"
+            },
+            "description": "OK"
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": ""
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
```

Comment:
These can be factored out to the top level. Putting them in each method adds noise.

Reply:
Thanks, moved it to the top and removed `produces` and `consumes` from each method.